### PR TITLE
request rider screen

### PR DESCRIPTION
**Pull request by Afinjuomo**
please kindly review

### CODE REVIEW

File: apps/Client/tailwind.config.js [hunk 1]
```diff
@@ -2,4 +2,14 @@
 export const content = [
   "./src/**/*.{js,ts,jsx,tsx}",
   "./public/index.html",
+  // Add other paths as needed
 ];
+export const theme = {
+  extend: {
+    colors: {
+      orange: {
+        500: '#ff9900', // Adjust the hex color code as needed
```
Comment (Don-christo) [inline apps/Client/tailwind.config.js:11]: Why the comment? You don't need to adjust the hex color. Find the one suitable and fix there, and remove the comment. The comment is not needed.

File: apps/Client/src/pages/RequestRiderPage.tsx
Comment (Don-christo) [inline apps/Client/src/pages/RequestRiderPage.tsx:1]: Making use of react-hook-form should be better than the normal form tag that comes with html.

**Review by Don-christo — CHANGES_REQUESTED** [review]

**Review by Afinjuomo — COMMENTED** [review]
updated to react hook form

**Comment by Afinjuomo** [comment]
Screenshot 2024-01-30 at 10 57 08 AM

**Review by Don-christo — APPROVED** [review]
Good job. LGTM!

**Pull request merged**